examples/lock-app/mbed/mbed_app.json

```diff
@@ -30,7 +30,7 @@
                 "CORDIO"
             ],
             "led-active-state": 1,
-            "lock-button": "USER_BUTTON",
+            "lock-button": "BUTTON1",
             "function-button": "NC"
         },
         "CY8CPROTO_062_4343W": {
```

Comment:
How this change is related with switching mbed to newer version?

Reply:
They changed it in new Mbed